src/main/scala/com/atomist/rug/runtime/js/migrations/GraphNodeFunctionsToProperties.scala

```diff
@@ -0,0 +1,5 @@
+package com.atomist.rug.runtime.js.migrations
+
+object NodeNameToProperty extends LiteralJavaScriptReplacementMigration("GraphNode.nodeName is now a property, not a function", "nodeName()", "nodeName")
```

Comment:
concern: This will run on all rugs forever. It's quite feasible that someday someone will have a function nodeName() somewhere. This will change it, and that's confusing.

Let's check the version of TS Modules they have, and only run this migration if they're out of date.